src/main/java/org/folio/circulation/domain/UpdateRequestQueue.java

```diff
@@ -48,10 +56,31 @@ public static UpdateRequestQueue using(Clients clients) {
 
       String requestPickupServicePointId = firstRequest.getPickupServicePointId();
 
-      if (checkInServicePointId == null || requestPickupServicePointId == null || checkInServicePointId.equalsIgnoreCase(requestPickupServicePointId)) {
+      if (checkInServicePointId.equalsIgnoreCase(requestPickupServicePointId)) {
```

Comment:
Again, NULL check is not needed here?

Reply:
When an item is checked in a service point is required. The `null` check was due to tests that did not set the service point on check in. I've corrected those tests, so the `checkInServicePointId` should never be `null` here.